registry/finders.go

```diff
@@ -96,17 +96,24 @@ func FindVersion(c *Space, appSlug, version string) (*Version, error) {
 		return nil, ErrVersionInvalid
 	}
 
-	db := c.VersDB()
-	row := db.Get(ctx, getVersionID(appSlug, version))
+	// We must try released and pending dbs
+	for _, db := range []*kivik.DB{c.VersDB(), c.PendingVersDB()} {
```

Comment:
We should probably not change this finder so that pending versions are not sent through the API until they are validated.

Reply:
So will create a new finder.
We need a way to find if uploading version is existing or not (pending + released) but also a way to find existing version (released)